packages/extension/src/components/atoms/TruncatedText/TruncatedText.tsx

```diff
@@ -0,0 +1,31 @@
+import { ComponentProps, FC } from 'react';
+
+import { Typography } from '@mui/material';
+
+type TruncatedTextProps = ComponentProps<typeof Typography> & {
+  text?: string;
+  maxLength?: number;
+};
+
+export const TruncatedText: FC<TruncatedTextProps> = ({ text, maxLength, sx, ...rest }) => {
```

Comment:
See my suggested updates for that component in my sandbox:

https://codesandbox.io/p/sandbox/boring-heisenberg-qrrv9w